plugins/Diagnostics/Diagnostic/DiagnosticResult.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+/**
+ * The result of a diagnostic.
```

Comment:
Maybe we should mention `<br/>`'s will be replaced by newlines in diagnostics:run but `<p>` won't? Things like that seem important to me.

Also: fixed the HTML in my original comment.